Modules/III-k6-Intermediate/08-Setting-load-profiles-with-executors/Ramping-Arrival-Rate-Exercises.md

```diff
@@ -46,110 +46,125 @@ k6 run test.js
+     http_reqs......................: 315    10.474815/s
+     iteration_duration.............: avg=119.81ms min=100.54ms med=112.83ms max=423.76ms p(90)=140.04ms p(95)=154.62ms
+     iterations.....................: 315    10.474815/s
+     vus............................: 2      min=2       max=2
+     vus_max........................: 2      min=2       max=2
 ```
 
 While _successful_, a closer inspection of our results shows that our test behavior was not as intended.
 
 Looking at the output, we see the following:
 
 ```bash
-WARN[0015] Insufficient VUs, reached 5 active VUs and cannot initialize more  executor=ramping-arrival-rate scenario=k6_workshop
+WARN[0015] Insufficient VUs, reached 2 active VUs and cannot initialize more  executor=ramping-arrival-rate scenario=k6_workshop
 ```
 
-What happened? With the `preAllocatedVUs` setting we glossed over earlier, we told k6 to start the test with 5 virtual users; after a few iterations, k6 was able to determine that it would **not** be able to achieve our desired iteration rate within the stage. This fact is evident by looking at the `iterations` value in the test summary; our test was only able to attain a rate of 3.32 iterations per second---we wanted 10.
+What happened? 
+
+With the `preAllocatedVUs` setting we glossed over earlier, we told k6 to start the test with 2 virtual users. With this pre-allocation, k6 could **not** achieve the desired iteration rate within the stage. 
+
+This fact is evident by looking at the `iterations` value in the test summary; our test was only able to attain a rate of 10.47 iterations per second and we wanted 30.
 
-We _could_ simply double the number of `preAllocatedVUs`, or better yet, we could allow k6 to automatically control the number of VUs to achieve our desired rate.
+### Adjusting preallocated virtual users
 
-### Autoscaling of virtual users
+Restating once again, the _Ramping Arrival Rate_ executor is focused on the _iteration rate_ over a time frame within each stage. k6 aims to eliminate the need to be overly concerned about the actual number of users required to achieve such a rate. However, we still need to give our script enough VUs to achieve that rate.
 
-Restating once again, the _Ramping Arrival Rate_ executor is focused on the _iteration rate_ over a period of time within stages. k6 aims to eliminate the need to be overly concerned about the actual number of users required to achieve such a rate. As a user of the executor, our script can simply specify the maximum number of VUs allowed, letting k6 handle the actual details. This can be referred to as _autoscaling_.
+From our example above, we have that our request duration or latency is, on average, 116.57ms, and the 95 percentile is around 151.35ms. With just 2 VUs (`preAllocatedVUs`), in a very optimistic scenario, we cannot expect much more than `2 VUs / 0.116 s = 17.24 iterations/s`. Even if this was the only factor at play, which we see is not in the next section.
```

Comment:
> From our example above, we have that our request duration or latency is, on average

The "request duration or latency is" part feels like it should be "request duration--or latency--is." 🤷 

> Even if this was the only factor at play, which we see is not in the next section.

This sentence seems incomplete.